packages/react-utils/src/createRequiredContext.ts

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+
+/**
+ * Create common context-related values from React with a required context.
```

Comment:
todo: I would try and rewrite this description somewhat as it's not really setting "context values", but rather pre-configures the context hook to require the Provider to be accessible, in turn ensuring the returned context value from the hook is always typed as intended.